deltacat/tests/compute/test_compact_partition_incremental.py

```diff
@@ -235,8 +238,22 @@ def test_compact_partition_incremental(
             "sort_keys": sort_keys if sort_keys else None,
         }
     )
+
     # execute
-    rcf_file_s3_uri = compact_partition_func(compact_partition_params)
+    def _incremental_compaction_setup():
+        """
+        This callable runs right before invoking the benchmark target function.
```

Comment:
Nit: Could add to this comment stating why you are not using a function scoped fixture to cleanup the RCF between test runs

Reply:
Sounds good